static/js/message_list_view.js

```diff
@@ -554,6 +554,24 @@ MessageListView.prototype = {
             }
         });
 
+        if (row.find('.message_content span.timestamp').length > 0) {
+            row.find('.message_content span.timestamp').each(function (index, elem) {
+                // Populate each timestamp span with mentioned time
+                // in user's local timezone.
+                var timestamp = moment.unix(elem.getAttribute('value'));
+                if (timestamp.isValid() && elem.getAttribute('value') !== null) {
+                    var rendered_time = timerender.render_timestamp(timestamp,
+                                                                    null,
+                                                                    elem.innerHTML);
+                    elem.innerHTML = rendered_time.text;
```

Comment:
Can these `innerHTML`s be accessing the text attribute instead?  It'd be more obviously not a potential source of XSS issues (we try to avoid using `innerHTML` unless absolutely necessary, and I think the content here is always plain text).